src/utils/authority.js

```diff
@@ -1,7 +1,7 @@
 // use localStorage to store the authority info, which might be sent from server in actual project.
 export function getAuthority() {
   // return localStorage.getItem('antd-pro-authority') || ['admin', 'user'];
-  return localStorage.getItem('antd-pro-authority') || 'admin';
+  return localStorage.getItem('antd-pro-authority') || 'admin1';
```

Comment:
We can remove `authority.js` in addition to the `Authorized.js` definition as this is not being used for managing users.

Reply:
@gurbirkalsi when I removed these two files, `BasicLayout.js` was throwing error as it is referenced in many places there. Do we want to remove the references from there as well?

Reply:
Yes, lets remove the references throughout the codebase!